status/kube.sh

```diff
@@ -0,0 +1,16 @@
+# Requires https://github.com/jonmosco/kube-tmux
+
+show_kube() {
+  local index icon color text  context_color namespace_color module
+
+  index=$1
+  icon=$(get_tmux_option "@catppuccin_kube_icon" "󱃾")
+  color=$(get_tmux_option "@catppuccin_kube_color" "$thm_blue")
+  context_color=$(get_tmux_option "@catppuccin_kube_context_color" "red")
+  namespace_color=$(get_tmux_option "@catppuccin_kube_namespace_color" "cyan")
+  text=$(get_tmux_option "@catppuccin_kube_text" "#(KUBE_TMUX_SYMBOL_ENABLE=false /bin/bash ${TMUX_PLUGIN_MANAGER_PATH}kube-tmux/kube.tmux 250 $context_color $namespace_color)")
```

Comment:
- `kube.tmux` is an executable no need to add `/bin/bash` would also not work on distro's like nixos.
- `${KUBE_TMUX_SYMBOL_ENABLE:=false}; ` to not overwrite what a user has already set.
  
  ![image](https://github.com/user-attachments/assets/b971ba4a-040c-48f5-a635-9debbdb20cf9)